browser/css/toolbar.css

```diff
@@ -1289,3 +1289,11 @@ html[dir='rtl'] #userListPopover::after {
 	font-size: var(--default-font-size);
 	color: var(--color-main-text);
 }
+.w2ui-overlay table.w2ui-drop-menu td.menu-text{
+	text-align: center;
+  	font-size: var(--toolbar-drop-menu-font-size);
```

Comment:
Please remove the space before the tab in this line

Reply:
I'm asking these because otherwise the lint test fails and it's impossible to build it